eventhubs/client/README.md

```diff
@@ -62,14 +62,46 @@ EventHubClient client = builder.build();
 
 ## Key concepts
 
-The concepts for AMQP are well documented in [OASIS Advanced Messaging Queuing Protocol (AMQP) Version
+- An **Event Hub client** is the primary interface for developers interacting with the Event Hubs client library, allowing for inspection of Event Hub metadata and providing a guided experience towards specific Event Hub operations such as the creation of producers and consumers.
```

Comment:
Can you run rewrap on this at 120 characters? It's hard to open markdown in code editors and have to use word wrap because these lines are so long.

https://marketplace.visualstudio.com/items?itemName=stkb.rewrap